### DISCUSSION TITLE
What about the -o option

### DISCUSSION THREAD

**Discussion by mgrossmann** [adesutherland/CREXX#207] [General]
Hi friends,

I'm not really happy with the current implementation of the file handling. (rxc / rxas)
Currently, I'm not able to use different directories for input and output. If I'm using
the -l option, this "working directory" will be prefixed to the output file.

Why do we need the -l option? Couldn't we simply use absolute paths/filenames?
Is this, because we prepend/append the file extension?

/Mike

**Comment by adesutherland** [comment]
The answer is super simple - it's because I hacked something together as quickly as possible! So any improvements are welcome - it would be good if we were consistent with gcc (?)

**Reply by mgrossmann** [reply]
> it would be good if we were consistent with gcc (?)

Yes, of course.

Thx :)

**Reply by rvjansen** [reply]
I think so too. My vote is also for -o and possibility for full paths on everything

**Reply by rvjansen** [reply]
also, have a look at #205
